pyfolio/timeseries.py

```diff
@@ -630,8 +630,28 @@ def rolling_fama_french(returns, factor_returns=None,
                         rolling_window=rolling_window)
 
 
-def perf_stats(returns, factor_returns=None, gross_lev=None):
-    """Calculates various performance metrics of a strategy, for use in
+def gross_lev(positions):
+    """
+    Calculates the gross leverage of a strategy.
+
+    Parameters
+    ----------
+    positions : pd.DataFrame
+        Daily net position values.
+         - See full explanation in tears.create_full_tear_sheet.
+
+    Returns
+    -------
+    pd.Series
+        Gross leverage.
+    """
+    exposure = positions.drop('cash', axis=1).abs().sum(axis=1)
+    return exposure/positions.sum(axis=1)
```

Comment:
Thinking about an example:
100% long, 100% short. lev should be 2. Is that the outcome here? Seems like positions.sum() is 0 in that case which would cause infinite leverage.